gallery-workflows/src/main/kotlin/com/r3/gallery/workflows/RevertEncumberedTokensFlow.kt

```diff
@@ -60,12 +61,16 @@ class RevertEncumberedTokensFlow(private val encumberedTxHash: SecureHash) : Flo
         }
 
         val outputStates = tokensStates.map { it.state.data.withNewHolder(encumberedTxIssuer) }
-
+        // enforce time window on the revert transaction to fall only after the time window to claim the encumbered tokens has expired
+        val timeWindowForRevert = TimeWindow.fromOnly(
+            lockState.state.data.timeWindow.untilTime!!.plusSeconds(31)
```

Comment:
@arnab-r3 any specific reason for +31 rather than +1?

Reply:
This is because the time window stored in the Lock state is that of the unsigned tx. The encumbrance tx adds 30s on top of it. Hence, the time window for reverting the encumbrance has to be greater than 30s. 

https://github.com/corda/gallery-demo/blob/main/gallery-workflows/src/main/kotlin/com/r3/gallery/workflows/OfferEncumberedTokensFlow.kt#L90